glide.yaml

```diff
@@ -0,0 +1,7 @@
+package: github.com/wavefronthq/wavefront-opentracing-sdk-go
+import:
+- package: github.com/opentracing/opentracing-go
+  version: ^1.0.2
+- package: github.com/wavefronthq/wavefront-sdk-go
+  version: ^0.9.0
```

Comment:
There is currently a deadlock bug in the wavefront-sdk-go when the proxy connection disconnects. I will have a fix by EOD. We should update this to point to the latest commit hash once my fix is in. Later on we can revert to using a particular release version.